app/controllers/course/courses_controller.rb

```diff
@@ -1,5 +1,6 @@
 class Course::CoursesController < Course::Controller
   def show #:nodoc:
+    @activities = recent_activities(current_course, 20)
```

Comment:
maybe function name is misleading... It should be `load_recent_activities` and what I thought is that we just need to load recent activities in course's show page.